app/scripts/views/settings/gravatar_permissions.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'views/form',
+  'stache!templates/settings/gravatar_permissions',
+  'lib/promise',
+  'views/mixins/back-mixin',
+  'views/mixins/service-mixin'
+],
+function (Cocktail, FormView, Template, p, BackMixin, ServiceMixin) {
+  'use strict';
+
+  var PERMISSIONS = ['profile:email'];
+  var GRAVATAR_MOCK_CLIENT_ID = 'gravatar';
+
+  var View = FormView.extend({
+    template: Template,
+    className: 'gravatar_permissions',
+
+    context: function () {
+      var account = this.getSignedInAccount();
+      var serviceName = 'Gravatar';
```

Comment:
Should this be the translated gravatar string that we have in the template {{#t}}Gravatar{{/t}} ?